odd_collector/adapters/kinesis/adapter.py

```diff
@@ -0,0 +1,42 @@
+import boto3
```

Comment:
AWS adapter must be in[odd-aws-collector](https://github.com/opendatadiscovery/odd-collector-aws)